configuration/configuration.go

```diff
@@ -345,6 +346,18 @@ func (gc *GlobalConfiguration) initTracing() {
 				log.Warn("Jaeger configuration will be ignored")
 				gc.Tracing.Jaeger = nil
 			}
+		case datadog.Name:
+			if gc.Tracing.DataDog == nil {
+				gc.Tracing.DataDog = &datadog.Config{
+					LocalAgentHostPort: "localhost:8126",
+					GlobalTag:          "",
+					Debug:              false,
+				}
+			}
+			if gc.Tracing.DataDog != nil {
```

Comment:
Should be 
```go
if gc.Tracing.Zipkin != nil {
	log.Warn("Zipkin configuration will be ignored")
	gc.Tracing.Zipkin = nil
}
if gc.Tracing.Jaeger != nil {
	log.Warn("Jaeger configuration will be ignored")
	gc.Tracing.Jaeger = nil
}
```

or 

```go
log.Warn("Zipkin & Jaeger configuration will be ignored")
gc.Tracing.Jaeger = nil
gc.Tracing.Zipkin = nil
```

According to the chosen solution, you also need to update the `Zipkin` and `Jaeger` case